jake-solution.js

```diff
@@ -0,0 +1,8 @@
+var num = 0;
+
+for (i = 0; i < 1000; i++) {
+  if (i % 3 == 0 || i % 5 == 0) {
```

Comment:
In javascript, it's safer to use triple equals (`===`) than double equals. Read up on it if you want, but as a general rule just use triple and you'll be fine.